samples/suit/flash_companion/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	chosen {
+		extmem-device = &mx25uw63;
```

Comment:
Since the flash companion is supposed to be a core image down the road should we mention how to set this alias in the main application devicetree overlay?